pkg/karmadactl/cmdinit/kubernetes/node.go

```diff
@@ -108,6 +109,10 @@ func (i *CommandInitOption) AddNodeSelectorLabels() error {
 		}
 	}
 
+	if etcdSelectorLabels == nil {
+		return errors.New("have no node can be scheduled")
+	}
```

Comment:
The message `have no node can be scheduled` might not be straightforward enough for the user to understand what's happening. 
How about changing it to `failed to find a healthy node for karmada-etcd`?

Reply:
Make sense.